src/mgmt/cli_mgmt.c

```diff
@@ -352,8 +382,10 @@ get_attach_info(const char *name, int *npsrs, struct dc_mgmt_psr **psrs,
 		rc = -DER_MISC;
 		goto out_dresp;
 	}
-	resp = mgmt__get_attach_info_resp__unpack(NULL, dresp->body.len,
+	resp = mgmt__get_attach_info_resp__unpack(&alloc.alloc, dresp->body.len,
 						  dresp->body.data);
+	if (alloc.oom)
+		D_GOTO(out_dresp, rc = -DER_MISC);
```

Comment:
a while ago I was asked not to add new D_GOTO()s, has this advice been reversed @liw @gnailzenh

Reply:
I've never been asked that, I've questioned the benefit of D_GOTO() but as far as I'm aware they're still the preferred style.